cranelift/codegen/src/isa/x64/inst/mod.rs

```diff
@@ -709,6 +716,11 @@ impl Inst {
         }
     }
 
+    pub(crate) fn xmm_rmi_reg(opcode: SseOpcode, src: RegMemImm, dst: Writable<Reg>) -> Inst {
```

Comment:
Is the lowering missing for these operations?

Reply:
Nope, see shifts.